apps/redi-talent-pool/src/components/organisms/job-listing-card/JobListingCard.scss

```diff
@@ -1,9 +1,5 @@
 @import '_variables.scss';
 
-.job-posting-link:hover {
-  text-decoration: none;
-}
-
```

Comment:
I think we need to keep this. Otherwise, every line of text gets underlined on hover.
![image](https://github.com/talent-connect/connect/assets/51786805/f2c8a6cd-e493-44ac-9710-07e31eb4559a)

Reply:
Good catch!